modules/shortcodes/js/jetpack-email-subscribe.js

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
I am not sure I understand the simplicity argument, since

a) the code is minified, so the filesize is not an issue
b) using jQuery is explicit and there is no confusion for what it stands for. After longer work with vanilla JavaScript in chrome, $ starts being remembered last-viewed-dom-element and not jquery
c) It is clear at every point that we are not going to fall into jQuery conflict trap, since we are not using shorthand

Reply:
👍 	That is really minor so more nit-picking than anything else. We tend to alias jQuery in other Jetpack files but that is not a must.